tensorflow_lite_support/web/tflite_model_runner/demo/dist/script.js

```diff
@@ -1,117 +1,48 @@
+const worker = new Worker("worker.js");
```

Comment:
Please also copy the worker.js to `dist` dir.

Reply:
Sorry, please ignore above comment. I found the worker.js has already existed in dist.